config/_default/config.toml

```diff
@@ -199,9 +199,9 @@ weight = 4
 
 [[languages.en.menu.main]]
 identifier = "acknowledgments"
-name = "Ack"
+name = "Support"
 url = "/acknowledgments/"
-weight = 70
+weight = 50
```

Comment:
Why did you update the weight?  What does this do?

Reply:
If you don't know what it does then don't change it

Reply:
Well, it's an educated guess, but it does make sense to try the original setting before changing.  If we end up with the crowded headings, then we can revisit.

Reply:
As far as I can tell it controls the order in which they appear, not the size of the font.

Reply:
After looking at it, I agree that "weight" controls the position on the page.